index.js

```diff
@@ -94,12 +94,19 @@ const path = require ('path');
 //.
 //. Indicates that this middleware does not form a response. The supplied value
 //. will be assigned to `res.locals` and the next middleware will be called.
+//.
+//# Render :: (String, Object) -> Response a
+//. Indicates a HTML response. The first argument will be the file path of the
+//. view file to render. This can be an absolute path, or a path relative to
+//. the Express views setting. The second argument indicates an object whose
+//. properties define local variables for the view.
 const Response = daggy.taggedSum ('Response', {
   Stream: ['code', 'mime', 'stream'],
   Json: ['code', 'value'],
   Redirect: ['code', 'url'],
   Empty: [],
   Next: ['locals'],
+  Render: ['view', 'data'],
```

Comment:
Could you move this below the `Json` constructor definition?

Also, I think it would be good to add a `code` parameter. That would allow users to use this to render custom 404 pages, for example.